docs/source/top_level_elements/name.rst

```diff
@@ -0,0 +1,350 @@
+====
+name
+====
+
+-----
+About
+-----
+
+This section describes our use of the top level element name.
+
+-------------------
+Names and Roleterms
+-------------------
+
+A Bit of Background and the Importance of Derefenceable Marcrelators
+====================================================================
+
+In rdf, both the Samvera and Islandora communities recommend marcrelators to serve as the rdf property that relate names
+to the digital object record.
+
+We usually use `mods:name/mods:role/mods:roleTerm` to store a marcrelator text value, but not always.  Here is the unique
+list of values we use for `mods:name/mods:role/mods:roleTerm`:
+
+.. code-block:: python
+
+    ['Addressee', 'Architect', 'Arranger', 'Artist', 'Associated', 'Associated name', 'Attributed', 'Attributed name', 'Author', 'Autographer', 'Cartographer', 'Choreographer', 'Client', 'Compiler', 'Composer', 'Contractor', 'Contributor', 'Copyright holder', 'Correspondent', 'Costume designer', 'Creator', 'Dedicatee', 'Depicted', 'Designer', 'Distributor', 'Donor', 'Editor', 'Engraver', 'Former owner', 'Honoree', 'Illustrator', 'Instrumentalist', 'Interviewee', 'Interviewer', 'Issuing body', 'Lithographer', 'Lyricist', 'Minute', 'Music copyist', 'Musical director', 'Originator', 'Other', 'Owner', 'Performer', 'Photographer', 'Printer', 'Printer of plates', 'Producer', 'Production company', 'Publisher', 'Set designer', 'Signer', 'Stage director', 'Stage manager', 'Standards body', 'Surveyor', 'Videographer', 'Witness', 'creator']
+
+Similarly, here are the text values related to marc relators:
+
+.. code-block:: python
```

Comment:
I'm not sure why these lists need to be in a python code block.

Reply:
Because it's a list?